extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/client.ts

```diff
@@ -46,3 +46,5 @@ export function getHeaders({
     "Argocd-Project-Name": `${project}`,
   };
 }
+
+
```

Comment:
Please configure your editor to stop adding extra new lines at the end of the file.

Reply:
sure Leo.